scripts/src/modules/search/mobile-filter-expander-enhanced.js

```diff
@@ -1,5 +1,5 @@
 /* global $buttonHtml $noOfFilters */
-
+alert("called");
```

Comment:
Could we please remove this before merging?

Reply:
* embarrassing debug oversight* - now removed ...